.github/codecov.yaml

```diff
@@ -2,19 +2,28 @@
 codecov:
   bot: "Codecov Bot"
   max_report_age: 12
-  require_ci_to_pass: yes
+  require_ci_to_pass: true
   notify:
     after_n_builds: 1
-    wait_for_ci: yes
+    wait_for_ci: true
+
+coverage:
+  status:
+    project:
+      default:
+        threshold: 1%
```

Comment:
cc @andrewdavidmackenzie 

Allow the coverage to drop by X%, and posting a success status.

Reply:
BTW new patch coverage raised up to [79.71%](https://app.codecov.io/gh/Kuadrant/kuadrant-operator/pull/260)

Reply:
Sounds good. Variability in test sampling can cause the % to vary a little (you can also specify the number of digits of rounding to do before comparing BTW), causing it to drop a little (with same code and tests) and fail - so this guards against that.

Reply:
e.g.
```
coverage:
  round: up
  precision: 0
```

Reply:
TIL: codecov [target](https://docs.codecov.com/docs/commit-status#target)
```
target

auto | <number>
Choose a minimum coverage ratio that the commit must meet to be considered a success.

    auto will use the coverage from the base commit (pull request base or parent commit) coverage to compare against.
    <number> you can specify a target of an exact coverage number such as 75% or 100% (string, int, or float accepted).
```